src/index.js

```diff
@@ -4,9 +4,10 @@ import delve from 'dlv';
  *	@param {Component} component	The component whose state should be updated
  *	@param {string} key				A dot-notated key path to update in the component's state
  *	@param {string} eventPath		A dot-notated key path to the value that should be retrieved from the Event or component
+ *	@param {function} cb			A function to be called once component state is updated
  *	@returns {function} linkedStateHandler
  */
-export default function linkState(component, key, eventPath) {
+export default function linkState(component, key, eventPath, cb) {
```

Comment:
Only suggestion I'd make would be to use the full word `callback` here.  It gets minified out anyway, and it's just that little bit more understandable for newcomers! 😊